packages/frontend/src/components/MkMediaAudio.vue

```diff
@@ -217,17 +217,29 @@ function showMenu(ev: MouseEvent) {
 		});
 	}
 
+	const details: MenuItem[] = [];
 	if ($i?.id === props.audio.userId) {
-		menu.push({
-			type: 'divider',
-		}, {
+		details.push({
 			type: 'link',
 			text: i18n.ts._fileViewer.title,
 			icon: 'ti ti-info-circle',
 			to: `/my/drive/file/${props.audio.id}`,
 		});
 	}
 
+	if (iAmModerator) {
+		details.push({
+			type: 'link',
+			text: i18n.ts._fileViewer.titleAdmin,
+			icon: 'ti ti-info-circle',
```

Comment:
ユーザーのメニューと揃えて

```suggestion
			text: i18n.ts.moderation,
			icon: 'ti ti-photo-exclamation',
```

でよさそう

Reply:
fixed 
b882a2e